api/Window.json

```diff
@@ -2829,6 +2865,42 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "relative-multi-screen": {
+          "__compat": {
+            "description": "Moved relative to the <a href='https://developer.mozilla.org/docs/Web/API/Window_Management_API/Multi-screen_origin'>Multi-screen origin</a>",
+            "support": {
+              "chrome": {
+                "version_added": "100"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1",
+                "notes": "Multi-screen coordinates are used, but windows are clamped to their existing displays."
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "1",
```

Comment:
Is it possible to leave this marked false with the note? Otherwise, this is probably okay.
Ditto for all other Safari entries.

Reply:
That doesn't sound correct to me — if it isn't supported at all, how can it exhibit inconsistent/unreliable behavior?

Reply:
Understood, this framing is reasonable.